step-functions/step-functions-handler/src/main/java/step/functions/handler/FunctionMessageHandler.java

```diff
@@ -68,6 +69,8 @@ public class FunctionMessageHandler extends AbstractMessageHandler {
 	private final ObjectMapper mapper;
 
 	private ExecutorService liveReportingExecutor;
+	// This is actually a Jakarta WebsocketContainer, but instantiated dynamically using a separate class loader
+	private volatile Object webSocketContainer;

```

Comment:
This could be wrapped in a class of the streaming api in order not to work with Object

Reply:
I don't currently see a good way of implementing this without overcomplicating things, but I'm open to suggestions.